syntax = "proto3";

import "google/protobuf/descriptor.proto";

package greetings;

message GreetingRequest {
  string name = 1;
}

message GreetingResponse {
  string message = 1;
}


service GreetingService {
  /* Say hello */
  rpc SayHello(greetings.GreetingRequest) returns (greetings.GreetingResponse) {}
  rpc DoFail(greetings.GreetingRequest) returns (greetings.GreetingResponse) {}
}
